lib/plugin/screenshotOnFail.js

```diff
@@ -69,7 +69,11 @@ module.exports = function (config) {
     }
   }
 
-  if (Codeceptjs.config.get().mocha) options.reportDir = Codeceptjs.config.get().mocha.reporterOptions.reportDir;
+  if (Codeceptjs.container.mocha()) {
+    options.reportDir = Codeceptjs.container.mocha().options.reporterOptions
+      && Codeceptjs.container.mocha().options.reporterOptions.reportDir;
```

Comment:
Mocha 4 used `options = options || {};` and changed options object directly.

Mocha 6 does use `options = utils.assign({}, mocharc, options || {});` and changes only local copy of options.
So it's not correctly to check param in config, when it does not changes and mocha have actual config in itslef

Reply:
Ok, then remove this